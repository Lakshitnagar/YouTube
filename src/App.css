.App {
  text-align: center;
}

.App-header {
  display: flex;
  justify-content: space-between;
  padding: 0 16px;
  height: 56px;
  border: solid 1px;
}
