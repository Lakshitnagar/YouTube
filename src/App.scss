.app {
  height: 100%;
  display: flex;
  flex-direction: column;
  align-items: stretch;
  font-family: Roboto, Arial, sans-serif;
}
