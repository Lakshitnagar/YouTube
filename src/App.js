import React from 'react';
import './App.scss';
import AppHeader from "./components/AppHeader/AppHeader";
import PopularSearches from "./components/PopularSearches/PopularSearches";
import SearchResult from "./components/SearchResult/SearchResult";

function App() {
    return (
        <div className="app">
            <header>
                <AppHeader/>
            </header>

            <PopularSearches/>

            <SearchResult/>
        </div>
    );
}

export default App;
