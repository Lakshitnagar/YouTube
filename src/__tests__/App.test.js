import React from 'react';
import App from "../App";
import {shallow} from "enzyme";
import {shallowToJson} from "enzyme-to-json";

describe('App', function () {
    it('should render and match snapshot', function () {
        const component = <App />;

        const wrapper = shallow(component);

        expect(shallowToJson(wrapper.childAt(0))).toMatchSnapshot();
    });
});
