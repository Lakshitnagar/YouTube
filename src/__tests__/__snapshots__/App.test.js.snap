// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render and match snapshot 1`] = `
<div
  className="app"
>
  <header>
    <AppHeader />
  </header>
  <PopularSearches />
  <SearchResult />
</div>
`;
